notifier/settings.py

```diff
@@ -65,14 +72,14 @@
 EMAIL_REWRITE_RECIPIENT = os.getenv('EMAIL_REWRITE_RECIPIENT')
 
 # LMS links, images, etc
-LMS_URL_BASE = os.getenv('LMS_URL_BASE', 'http://localhost:8000')
+LMS_URL_BASE = os.getenv('LMS_URL_BASE', 'http://localhost:18000')
```

Comment:
Why we are changing the port here and at L82 for default values? I believe these values are coming from configuration and should work in devstack without changing them.